proton-c/bindings/cpp/README.md

```diff
@@ -0,0 +1,24 @@
+# C++ binding for proton.
+
+This is a C++ wrapper for the proton reactor API.
```

Comment:
It's not a "wrapper for the proton reactor" API it's a "C++ binding for the Proton API"